components/VotePanel/CastMultiVoteButtons.tsx

```diff
@@ -0,0 +1,198 @@
+          status = status.map(() => false);
+          status[index] = true;
+          options = [index];
+        } else {
+          // remove nota from the selected if any other option is clicked
+          status[last] = false;
+          options = options.filter(option => option !== last);
+          if (!options.includes(index)) {
+            options.push(index)
+          }
+          status[index] = true;
+        }
+      } else {
+        if (!options.includes(index)) {
+          options.push(index)
+        }
+        status[index] = true;
+      }
+    } else {
+      options = options.filter(option => option !== index);
+      status[index] = false
+    }
+
+    setSelectedOptions(options);
+    setOptionStatus(status);
+  }
+
+  return (isVoting && !isVoteCast) ? (
+    <div className="bg-bkg-2 p-4 md:p-6 rounded-lg space-y-4">
+      <div className="flex flex-col items-center justify-center">
+        <h3 className="text-center">Cast your {votingPop} vote</h3>
+      </div>
+
+      <div className="items-center justify-center flex w-full gap-5">
+        <div
+          className={`w-full flex flex-col justify-between items-center gap-3`}
+        >
+          {proposal.options.map((option, index) => {
+            return (
+            <div className="w-full" key={index}>
+              {optionStatus[index] 
+              ?
+                <Button
+                  tooltipMessage={tooltipContent}
+                  className='w-full bg-fgd-1'
+                  onClick={() => handleOption(index, false)}
+                  disabled={!canVote || multiChoiceSubmitting}
+                  isLoading={multiChoiceSubmitting}
+                >
+                  {option.label}
+                </Button> 
+              :
+                <SecondaryButton
+                tooltipMessage={tooltipContent}
+                className='w-full'
+                onClick={() => handleOption(index, true)}
+                disabled={!canVote || multiChoiceSubmitting}
+                isLoading={multiChoiceSubmitting}
+              >
+                {option.label}
+              </SecondaryButton>
+              }  
+            </div>
+            )}
+          )}
+          <div className="text-sm">
+            Note: You can select one or more options
+          </div>
+          <Button
+            tooltipMessage={
+              tooltipContent === "" && !selectedOptions.length ? 
+              `Select at least one option to vote` 
+              : tooltipContent
+            }
+            className="w-full"
+            onClick={() => handleVote('yes')}
+            disabled={!canVote || multiChoiceSubmitting || !selectedOptions.length}
+            isLoading={multiChoiceSubmitting}
+          >
+            <div className="flex flex-row items-center justify-center">
+              Vote
+            </div>
+          </Button>
+        </div>
+      </div>
+
+      {showVoteModal && vote ? (
```

Comment:
TODO use flag here to hide this until bug is fixed

Reply:
Bug is fixed so didn't change anything.